app/controllers/chargeback_report_controller.rb

```diff
@@ -5,71 +5,46 @@ class ChargebackReportController < ApplicationController
   after_action :set_session_data
 
   include Mixins::SavedReportPaging
+  include Mixins::GenericListMixin
   include Mixins::GenericSessionMixin
+  include Mixins::GenericShowMixin
   include Mixins::BreadcrumbsMixin
 
   def self.table_name
-    @table_name ||= "chargeback_report"
+    @table_name ||= "miq_report_result"
```

Comment:
I'm not entirely sure that this is a good idea, I'm not that familiar with reporting, but this table seems too generic to me.